apps/schematic/api/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tiangolo/uwsgi-nginx-flask:python3.10
+FROM tiangolo/uwsgi-nginx-flask:python3.10-2024-09-16
```

Comment:
Thanks for the change @tschaffter . I could pre-approve for now. I am curious why you selected `python3.10-2024-09-16`. Is it because this keeps the underlying Python version the same? 

Also, I am still seeing that in the base image of `tiangolo/uwsgi-nginx-flask:python3.10` [here](https://github.com/tiangolo/uwsgi-nginx-flask-docker/blob/master/docker-images/python3.10.dockerfile), they are pulling from the docker image [here](https://github.com/tiangolo/uwsgi-nginx-docker/blob/master/docker-images/python3.10.dockerfile) which is still using `python:3.10-bullseye` as the base Python image.  Does that mean python updated `python:3.10-bullseye` to a different python version?

Reply:
These are great questions. Regarding the first one, my hypothesis was that the issue stemmed from a recent update to `tiangolo/uwsgi-nginx-flask:python3.10` and the fact that Schematic uses a moving tag. I then searched for a non-moving tag for `python3.10`, such as those that typically include a date. I selected the second most recent tag, which was built around a week ago (`python3.10-2024-09-16`). After rebuilding the image using this tag, the `schematic-api:build-image` task succeeded.

Yes, it's likely that `python:3.10-bullseye` is also a moving tag, which introduced changes to `tiangolo/uwsgi-nginx-flask:python3.10`.

Reply:
ooooh I see. Thank you so much! Using a moving tag seems to be a bad idea then. They are using a moving tag which caused their python version to change, and since we are also using a moving tag, that ends up affecting us.